stdlib/math/math.flux

```diff
@@ -0,0 +1,198 @@
+package math
+
+// builtin constants
+builtin pi
+builtin e
+builtin phi
+builtin sqrt2
+builtin sqrte
+builtin sqrtpi
+builtin sqrtphi
+builtin ln2
+builtin log2e
+builtin ln10
+builtin log10e
+builtin maxfloat
+builtin maxint
+builtin minint
+builtin maxuint
+
+//missing builtin values [y1 sqrte roundtoeven nextafter32 sqrtphi sqrtpi copysign ceil], extra builtin values [Ceil sqrtPhi sqrtPi sqrtE roundToEven copySign]
+// builtin functions
+builtin abs
+builtin acos
+builtin acosh
+builtin asin
+builtin asinh
+builtin atan
+builtin atan2
+builtin atanh
+builtin cbrt
+builtin ceil
+builtin copysign
+builtin cos
+builtin cosh
+builtin dim
+builtin erf
+builtin erfc
+builtin erfcinv
+builtin erfinv
+builtin exp
+builtin exp2
+builtin expm1
+builtin float64bits
+builtin float64frombits
+builtin floor
+builtin frexp
+builtin gamma
+builtin hypot
+builtin ilogb
+builtin m_inf
```

Comment:
is there a name we can use that doesn't use and `_`?

Reply:
sure.  it conflicts with the global namespace and we can rename it to `inf` when we resolve the scope correctly for `map` and `filter`